clang/runtime/dpct-rt/include/dpl_extras/dpcpp_extensions.h

```diff
@@ -399,6 +400,34 @@ template <typename T, int VALUES_PER_THREAD> class exchange {
     }
   }
 
+  /// Rearrange elements from blocked order to striped order
+  template <typename Item>
+  __dpct_inline__ void blocked_to_striped(Item item,
+                                          T (&keys)[VALUES_PER_THREAD],
+                                          int (&ranks)[VALUES_PER_THREAD]) {
+    T *buffer = reinterpret_cast<T *>(_local_memory);
+
+#pragma unroll
+    for (int i = 0; i < VALUES_PER_THREAD; i++) {
+      int offset = (item.get_local_id(0) * VALUES_PER_THREAD) + i;
+      if (INSERT_PADDING)
```

Comment:
Since `INSERT_PADDING` is a constexpr variable, could we make this an `if constexpr (...)`?

Reply:
Added this , thanks